src/quicklist.c

```diff
@@ -502,23 +510,48 @@ int quicklistNodeExceedsLimit(int fill, size_t new_sz, unsigned int new_count) {
     redis_unreachable();
 }
 
+/* Check if the limit of the quicklist node has been reached to determine if
+ * insertions, merges or other operations that would increase the size of
+ * the node can be performed.
+ * Return 1 if exceeds the limit, otherwise 0. */
```

Comment:
```suggestion
/* Determine whether the quicklist node has reached its limit before performing
 * insertions, merges, or other operations that would increase the node size.
 * Return 1 if exceeds the limit, otherwise 0. */
```